pylint_odoo/test/main.py

```diff
@@ -8,7 +8,7 @@
 from pylint_odoo import misc
 
 
-EXPECTED_ERRORS = 54
+EXPECTED_ERRORS = 57
```

Comment:
Add a case where the version is 10.*
https://github.com/vauxoo-dev/pylint-odoo/blob/dac51dad04a55b521c3e4ac0c2c3f357b3d6b912/pylint_odoo/test_repo/test_module/__openerp__.py#L6